docs/resources/samples/community-seasons-greetings.md

```diff
@@ -0,0 +1,773 @@
+      .getFormat()
+      .getFill()
+      .setColor("FFFFFF");
+    // Set fill color to FFFFFF for range HappyXmasTree!J17
+    selectedSheet.getRange("J17")
+      .getFormat()
+      .getFill()
+      .setColor("FFFFFF");
+  }
+
+  function FlashingStarAndSmileFF0000(workbook: ExcelScript.Workbook) {
+    //red
+    let selectedSheet = workbook.getWorksheet('HappyXmasTree');
+    // Set fill color to FF0000 for range HappyXmasTree!L2:L6
+    selectedSheet.getRange("L2:L6")
+      .getFormat()
+      .getFill()
+      .setColor("FF0000");
+    // Set fill color to FF0000 for range HappyXmasTree!K3:K5
+    selectedSheet.getRange("K3:K5")
+      .getFormat()
+      .getFill()
+      .setColor("FF0000");
+    // Set fill color to FF0000 for range HappyXmasTree!M3:M5
+    selectedSheet.getRange("M3:M5")
+      .getFormat()
+      .getFill()
+      .setColor("FF0000");
+    // Set fill color to FF0000 for range HappyXmasTree!N4
+    selectedSheet.getRange("N4")
+      .getFormat()
+      .getFill()
+      .setColor("FF0000");
+    // Set fill color to FF0000 for range HappyXmasTree!J4
+    selectedSheet.getRange("J4")
+      .getFormat()
+      .getFill()
+      .setColor("FF0000");
+    // Set fill color to 000000 for range HappyXmasTree!O26
+    selectedSheet.getRange("O26")
+      .getFormat()
+      .getFill()
+      .setColor("000000");
+    // Set fill color to 000000 for range HappyXmasTree!I26
+    selectedSheet.getRange("I26")
+      .getFormat()
+      .getFill()
+      .setColor("000000") //black
+  }
+
+  function FlashingStarAndSmileFFFF00(workbook: ExcelScript.Workbook) {
+    //yellow
+    let selectedSheet = workbook.getWorksheet('HappyXmasTree');
+    // Set fill color to FF0000 for range HappyXmasTree!L2:L6
+    selectedSheet.getRange("L2:L6")
+      .getFormat()
+      .getFill()
+      .setColor("FFFF00");
+    // Set fill color to FF0000 for range HappyXmasTree!K3:K5
+    selectedSheet.getRange("K3:K5")
+      .getFormat()
+      .getFill()
+      .setColor("FFFF00");
+    // Set fill color to FF0000 for range HappyXmasTree!M3:M5
+    selectedSheet.getRange("M3:M5")
+      .getFormat()
+      .getFill()
+      .setColor("FFFF00");
+    // Set fill color to FF0000 for range HappyXmasTree!N4
+    selectedSheet.getRange("N4")
+      .getFormat()
+      .getFill()
+      .setColor("FFFF00");
+    // Set fill color to FF0000 for range HappyXmasTree!J4
+    selectedSheet.getRange("J4")
+      .getFormat()
+      .getFill()
+      .setColor("FFFF00");
+    // Set fill color to 000000 for range HappyXmasTree!O26
+    selectedSheet.getRange("O26")
+      .getFormat()
+      .getFill().clear
+      //.setColor("000000");
+    // Set fill color to 000000 for range HappyXmasTree!I26
+    selectedSheet.getRange("I26")
+      .getFormat()
+      .getFill().clear
+      //.setColor("000000")
+  }
+
+  //OuterEdgeClearFill()
+  console.log('Routine finished')
+  }
+
+function OuterEdgeFFFF00(workbook: ExcelScript.Workbook) {
```

Comment:
I think we can rewrite this to use RangeAreas and save so much space. Am I right here, @sumurthy?

Reply:
Tracking with [feature](https://office.visualstudio.com/OC/_workitems/edit/4924924)